### PR TITLE
(node/lsstcam-ccobtb.cp.lsst.org) Deploy host as EL9. 

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

File: hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.83/26,139.229.175.126"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:12]: image
Missing device association.

File: hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    content:
+      connection:
+        id: "enp6s0"
+        uuid: "da97db8e-6400-4cca-9334-71995eb2ac1d"
+        type: "ethernet"
+        interface-name: "enp6s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.0.200/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:29]: What does this interface connect to?

File: hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    content:
+      connection:
+        id: "enp7s0"
+        uuid: "da97db8e-6400-4cca-9334-71995eb2ac1d"
+        type: "ethernet"
+        interface-name: "enp7s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.2.1/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:43]: What does this interface connect to?

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:12]: Device is associated on ipam.cp.lsst.org

File: hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml (hunk 2 above)
Comment (glennmorris) [reply hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:29]: > What does this interface connect to?

Either direct connection to some device, or via a private switch. @tony-johnson could say specifically.

Ref:
https://lsstc.slack.com/archives/C06V270ECBW/p1716219458412489?thread_ts=1716210023.754989&cid=C06V270ECBW

File: hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml (hunk 3 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:43]: @tony-johnson  Do you know where this connects to ?

Comment (glennmorris) [reply hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml:43]: The best answer I can get is that these two attached devices are likely a PDU and some Keysight device.
Is that enough to merge this? It has been several weeks...

**Review by glennmorris — APPROVED** [review]

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**